scripts/coverage.sh

```diff
@@ -115,3 +123,9 @@ genhtml --output-directory target/cov/$reportName \
 )
 
 ls -l target/cov/$reportName/index.html
+ln -sfT $reportName target/cov/LATEST
+
+if [[ -n $failure_code ]]
+then
+  exit $failure_code
+fi
```

Comment:
Set `failure_code=0` on line 58 and then `exit $failure_code` here unconditionally?

Reply:
nice!